check_upgrade.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+VERSION=3.1.0.1971
+
+# stop current cluster
+./start-splice-cluster -k
+
+# clean up platform_it
+cd platform_it
+git clean -dfx
+cd ..
+
+# download the previous standalone data
+aws s3 cp s3://splice-snapshots/upgrade_tests/platform_it_${VERSION}.tar.gz .
+tar -xzvf platform_it_${VERSION}.tar.gz
+rm platform_it_${VERSION}.tar.gz
+
+# restart cluster
+./start-splice-cluster -l
```

Comment:
I would use `-b` here since we haven't changed any code

Reply:
i did a `git clean -dfx` in platform_it, so which removes jars from there, so i need to rebuild those.